test/DaprExtensionTests/Tests/DaprServiceInvocationTriggerTests.cs

```diff
@@ -157,7 +198,6 @@ public static string Sample([DaprServiceInvocationTrigger] JObject args, ILogger
                 [DaprServiceInvocationTrigger] JObject input,
                 [DaprState("store1", Key = "{input.stateKey}")] string existingState)
```

Comment:
How is this different from the GetState1 test?

Reply:
Sorry for not making this more clear in the PR. The difference is that the binding expression uses a more complex binding expression (`{input.stateKey}`) instead of a simple parameter name (`{stateKey}`).